generate_plots.py

```diff
@@ -506,6 +507,172 @@ def create_treemap():
+                    <div class="arrowhead"></div>
+                    """
+            )
+
+            for i, third_key in enumerate(reversed(component_tree[second_key])):
+                third_color = color_map.get(components[third_key]["assessment"], "")
+                if "plan" in components[third_key]["title"]:
+                    year = "Plan"
+                else:
+                    year = "".join(
+                        [x for x in components[third_key]["title"] if x.isnumeric()][
+                            0:4
+                        ]
+                    )
+
+                li_class = ""
+                if i == 3:
+                    li_class = "last"
+                elif i == 1:
+                    li_class = "timeline-inverted "
+
+                third_html_comp = (
+                    """
+                    <li class=" """
+                    + li_class
+                    + """ ">
+                        <div class="timeline-badge">"""
+                    + year
+                    + """ </div>
+                        <div class="timeline-panel """
+                    + third_color
+                    + """ ">
+
+                            <div class="timeline-heading">
+                                <h3 class="timeline-title">"""
+                    + components[third_key]["title"]
+                    + """</h3>
+                            </div>
+
+                            <div class="timeline-body">
+                                <p>
+                                    """
+                    + components[third_key]["text"]
+                    + """
+                                </p>
+                            </div>
+
+                        </div>
+                    </li>
+                """
+                )
+
+                if i == 3:
+                    third_html_comp = (
+                        '<div style="clear: both"></div>\n' + third_html_comp
+                    )
+
+                second_html_comp += third_html_comp
+
+            second_html_comp += "</div></ul></div>"
+
+            html_comp += second_html_comp
+
+        html_comp += "</div>"
+
+        html_acc += html_comp + "\n"
+
+    html_acc += """
+        <script>
+            var coll = document.getElementsByClassName("collapsible");
+            var i;
+
+            for (i = 0; i < coll.length; i++) {
+                coll[i].addEventListener("click", function() {
+                    this.classList.toggle("active");
+                    var content = this.nextElementSibling;
+                    if (content.style.display === "block") {
+                        content.style.display = "none";
+                    } else {
+                        content.style.display = "block";
+                    }
+                });
+            }
+        </script>
+        </div>
+    """
+
+    with open(
+        Path("hugo/layouts/shortcodes/modules_muenster.html"), "w", encoding="utf-8"
```

Comment:
Den Pfad bitte noch dynamisch anpassen (obwohl wir bisher nur für Münster solche Daten haben ...), s. entsprechende Zeilen in `create_treemap()`. Dann stellt sich mir noch die Frage, ob wir `create_treemap()` überhaupt noch brauchen?